code.js

```diff
@@ -1,3 +1,14 @@
 function fib(n) {
-    return n;
+    if(n==1) {
+        return 1;
+    }
+    else if(n==2) {
+        return 1;
+    }
+    else if(n==3) {
+        return 2;
+    }
+    else {
+        return fib(n-1) + fib(n-2) + n;
```

Comment:
You don't need to use two recursive calls. You can use the stored values in the array as a helper.